src/main/kotlin/org/rust/ide/annotator/RsErrorAnnotator.kt

```diff
@@ -1148,6 +1147,15 @@ private fun checkDuplicates(
     message.addToHolder(holder)
 }
 
+private fun checkConstGenerics(holder: RsAnnotationHolder, constParameter: RsConstParameter) {
+    if (CONST_GENERICS.availability(constParameter) == AVAILABLE
+        || MIN_CONST_GENERICS.availability(constParameter) == AVAILABLE) return
+    val current = constParameter.cargoProject?.rustcInfo?.version?.semver
+    val since = MIN_CONST_GENERICS.since
+    val feature = if (current != null && since != null && current >= since) MIN_CONST_GENERICS else CONST_GENERICS
+    feature.check(holder, constParameter, "const generics")
+}
```

Comment:
Note, `min_const_generics` feature is only about integers, `bool` and `char`. More complex options are still under `const_generics`. Also, `min_const_generics` and `const_generics` cannot be used together, so I think in case, when `min_const_generics` exists but code requires `const_generics`, we should replace `min_const_generics` with `const_generics`

For example, in the following case
```rust
#[derive(Eq, PartialEq)]
struct S(i32);
fn foo<const N: S>() {}
```

we should warn about `const_generics` instead of `min_const_generics`